src/psyclone/psyir/nodes/extract_node.py

```diff
@@ -68,11 +68,23 @@ class ExtractNode(PSyDataNode):
     :type parent: :py:class:`psyclone.psyir.nodes.Node`
     :param options: a dictionary with options provided via transformations.
     :type options: dictionary of string:values or None
+    :param str options["class"]: a prefix to use for the PSyData module name \
```

Comment:
In looking at the tests, I realised that (to me at least), 'class' is not a very intuitive name. Perhaps e.g. 'prefix' would be better?

Reply:
Done, see above.